core/domain/exp_jobs_one_off.py

```diff
@@ -476,3 +476,30 @@ def reduce(key, values):
             index += 2
         output_values.sort()
         yield (key, output_values)
+
+
+class RemoveTranslatorIdsOneOffJob(jobs.BaseMapReduceOneOffJobManager):
+    """Job that deletes the translator_ids from the ExpSummaryModel
+    """
+
+    @classmethod
+    def entity_classes_to_map_over(cls):
+        return [exp_models.ExpSummaryModel]
+
+    @staticmethod
+    def map(exp_summary_model):
+        if 'translator_ids' in exp_summary_model._properties:  # pylint: disable=protected-access
```

Comment:
Please add a comment here that is similar to https://github.com/oppia/oppia/blob/170bdeae5912ced0abc71257f5b0e5ca98fd1418/core/domain/activity_jobs_one_off.py#L167

Reply:
Done.